src/common/crypto/mbedtls/driverwrapper.cpp

```diff
@@ -213,11 +232,78 @@ static int ExportECPublicKeyToDER(
     return 0;
 }
 
+AlgorithmDescription RSASHAAlgorithm(size_t modulusBitlen)
```

Comment:
It is unclear why you always need to keep all three values if you can always get it. For example:
```c++
aos::RetWithError<psa_algorithm_t> GetPSAAlg(aos::crypto::HashEnum hash)
{
...
}

aos::RetWithError<mbedtls_md_type_t> GetMDType(aos::crypto::HashEnum hash)
{
...
}

aos::crypto::HashEnum GetRSASHAAlgorithm(size_t modulusBitlen)
{
...
}
```

Reply:
Done